libs/ui/src/theme/theme.ts

```diff
@@ -0,0 +1,24 @@
+import { sky, tomato, tomatoDark, skyDark } from '@radix-ui/colors';
```

Comment:
Do we want to consider using consistent camel casing for all theme values? 

`themeName` `bgColor` `fontColor` instead of PascalCase for the color items?